src/parser_urdf_TEST.cc

```diff
@@ -22,6 +22,20 @@
 #include "sdf/sdf.hh"
 #include "parser_urdf.hh"
 
+/////////////////////////////////////////////////
+/// Check that _a contains _b
+static bool contains(const std::string &_a, const std::string &_b)
+{
+  return _a.find(_b) != std::string::npos;
+}
```

Comment:
There are 4 copies of this  function new in various tests :). Can you just put it in `test_utils.h` as an `inline` function?

Reply:
https://github.com/gazebosim/sdformat/pull/1238/commits/cb6d540769af94cd41bcc6bc17de74d96deea39d

Reply:
There are uses of the same `contains` functions in `ign_TEST`, `parser_TEST`, `deprecated_specs` and `pose_1_9_sdf` from search, will have a quick refactor PR after this one to use the common ones in `test_utils`